website/docs/commands/environment-variables.html.md

```diff
@@ -100,3 +100,9 @@ The data directory is used to retain data that must persist from one command
 to the next, so it's important to have this variable set consistently throughout
 all of the Terraform workflow commands (starting with `terraform init`) or else
 Terraform may be unable to find providers, modules, and other artifacts.
+
+## TF_IN_AUTOMATION
+
+When the environment variable `TF_IN_AUTOMATION` is set to any non-empty value, Terraform makes some minor adjustments to its output to de-emphasize specific commands to run. The specific changes made will vary over time, but generally-speaking Terraform will consider this variable to indicate that there is some wrapping application that will help the user with the next step.
```

Comment:
Good catch! I went back and found the commit where this feature was added (0fe43c8977a04a64c2dff43aa1c36f6ba0a39e21 in the terraform repo), and it sounds like folks were worried about people depending too much on specific output changes. I think we can address that concern by just emphasizing that this is only about the human-readable output, and that the exact behavior is arbitrary.

```suggestion
If `TF_IN_AUTOMATION` is set to any non-empty value, Terraform adjusts its
output to avoid suggesting specific commands to run next. This can make the
output more consistent and less confusing in workflows where users don't
directly execute Terraform commands, like in CI systems or other wrapping
applications.

This is a purely cosmetic change to Terraform's human-readable output, and the
exact output differences can change between minor Terraform versions.
```